website/templates/website/create_normal.html

```diff
@@ -0,0 +1,132 @@
+{% extends "website/layout.html" %}
+{% block title %}Project Selection{% endblock %}
+{% block body %}
+<div class="container">
+    <div class="row"></div>
+    <div class="row">
+        <div class="col-md-12">
+            {% if heading %}
```

Comment:
why is this conditional? it looks like this template always passes in a heading. Also why are we passing it in instead of just coding it into the template directly?